modules/core/src/main/scala/org/scalasteward/core/application/StewardAlg.scala

```diff
@@ -44,7 +47,8 @@ final class StewardAlg[F[_]](config: Config)(implicit
     selfCheckAlg: SelfCheckAlg[F],
     streamCompiler: Stream.Compiler[F, F],
     workspaceAlg: WorkspaceAlg[F],
-    F: BracketThrow[F]
+    F: BracketThrow[F],
+    githubAppApiAlg: GitHubAppApiAlg[F]
 ) {
```

Comment:
Nitpick: I find it nicer if the `F` parameter comes last and everything else is sorted alphabetically.